HCCGo/app/js/jobSubmissionCtrl.js

```diff
@@ -145,8 +145,8 @@ jobSubmissionModule.controller('jobSubmissionCtrl', ['$scope', '$log', '$timeout
     $("#submitbtn").prop('disabled', true);
 
     // Separate SBATCH options from commands
-    job.commands = editor.getValue();
-    var other = editor.getValue().split(/\r?\n/);
+    job.commands = editor.getValue().replace(/\r\n/, "\n");
+    var other = job.commands.split(/\n/);
```

Comment:
These changes should probably go into a different pull request.  It's a separate issue.